inc/sainexthopgroup.h

```diff
@@ -74,6 +80,69 @@ typedef enum _sai_next_hop_group_member_observed_role_t
 
 } sai_next_hop_group_member_observed_role_t;
 
+/**
+ * @brief Attribute data for #SAI_NEXT_HOP_GROUP_MEMBER_ATTR_ADMIN_STATE
+ */
+typedef enum _sai_next_hop_group_member_admin_state_t
+{
+    /** Active */
+    SAI_NEXT_HOP_GROUP_MEMBER_ADMIN_STATE_ACTIVE,
+
+    /** Inactive. Graceful shutdown */
+    SAI_NEXT_HOP_GROUP_MEMBER_ADMIN_STATE_INACTIVE,
+
+    /** Force inactive. Disruptive to the connections */
+    SAI_NEXT_HOP_GROUP_MEMBER_ADMIN_STATE_FORCE_INACTIVE,
+
+} sai_next_hop_group_member_admin_state_t;
+
+/**
+ * @brief Attribute data for #SAI_NEXT_HOP_GROUP_MEMBER_ATTR_OPER_STATE
+ */
+typedef enum _sai_next_hop_group_member_oper_state_t
+{
+    /**
+     * @brief Inactive. Does not serve any connections and is safe to remove
+     */
+    SAI_NEXT_HOP_GROUP_MEMBER_OPER_STATE_INACTIVE,
+
+    /**
+     * @brief Pending becoming active. The new connections are not yet
+     * distributed to that member
+     */
+    SAI_NEXT_HOP_GROUP_MEMBER_OPER_STATE_ACTIVE_PENDING,
+
+    /**
+     * @brief Active. Included in the distribution of the new connections
+     */
+    SAI_NEXT_HOP_GROUP_MEMBER_OPER_STATE_ACTIVE,
+
+    /**
+     * @brief Pending becoming inactive. The new connections are no longer
+     * distributed to this member, but it still has some open connections,
+     * so it's not yet safe to remove.
+     */
+    SAI_NEXT_HOP_GROUP_MEMBER_OPER_STATE_INACTIVE_PENDING,
```

Comment:
in case of inactive pending, user would like to know how many connections are still on this member, so that user can decide whether to force inactive or not. without such status, if there is a single long tcp connection, then the group member can never be removed.